examples/oriented_images.js

```diff
@@ -0,0 +1,107 @@
+/* global itowns, document, renderer, initPicture  */
+// # Simple Globe viewer
```

Comment:
Could you adapt the title of the example :) ?

Reply:
yes